README.md

```diff
@@ -85,79 +85,90 @@ The task folder in this repo holds an existing site, including multiple html fil
     )
     ```
   4. We now need to make the relevant folders. Create a ```views``` directory within src. This is what you set in the app.js file as the directory to hold the hbs files.
-  5. In src/views create a ```layouts``` directory.
-  6. In src/views create a ```partials``` directory.
+  5. In `./src/views` create a ```layouts``` directory.
+  6. In `./src/views` create a ```partials``` directory.
 
 
 ### Serving the home page
 
-1. Create a ```home.hbs``` in the views directory
-1. Navigate to public/index.html. Copy all the contents to ```home.hbs```.
-1. Delete public/index.html.
-1. In src/views/layouts create a main.hbs file and insert the following code:
+1. Inside `./src/views/layouts`, create ```main.hbs```.
+2. Navigate to `./public/index.html`. Copy all the contents into ```main.hbs```.
+3. Delete `./public/index.html`.
+4. In `./src/views`, create a file called `home.hbs`.
+5. Inside `main.hbs`, select everything inside the ```<main>``` tag (not the `<main>` tag itself). Copy this and paste into `home.hbs`.  
+6. In `main.hbs`, delete everything inside `<main>` tag  and replace it with the following code:
     ```hbs
-    {{{ body }}} 
+    {{{ body }}}
     ```
     Body will be the hbs file we are rendering.
 
-6. To start the server in development mode use the npm script from the ```package.json``` by typing this in your terminal:
+7. To start the server in development mode use the npm script from the ```package.json``` by typing this in your terminal:
```

Comment:
Also wonder whether it's worth mentioning that this script 'watches' changes made to the server. 'Development mode' is a bit ambiguous.